helm/cluster-cloud-director/values.schema.json

```diff
@@ -264,7 +264,7 @@
                         "baseDomain",
                         "network"
                     ],
-                    "additionalProperties": false,
```

Comment:
I really don't like having to do this, but the alternative is to start duplicating parts of the `cluster` schema into this chart and that feels more hacky+messy.

See this thread for more info https://gigantic.slack.com/archives/C07KSM2E51A/p1727213002441369